src/vmprof_common.h

```diff
@@ -61,6 +67,36 @@ static int read_trace_from_cpy_frame(PyFrameObject *frame, void **result, int ma
 {
     int depth = 0;
 
+    if (profile_lines) {
```

Comment:
a comment here describing how the situation can be easily improved with storing lnotab in the dump instead of recomputing it each time